packages/http-framework-i18n/package.json

```diff
@@ -0,0 +1,59 @@
+{
+	"name": "@skyra/http-framework-i18n",
+	"version": "0.0.1",
+	"description": "The common pieces for ArchId Network's HTTP-only bots",
+	"author": "@skyra",
+	"license": "Apache-2.0",
+	"main": "dist/index.js",
+	"module": "dist/index.mjs",
+	"types": "dist/index.d.ts",
+	"exports": {
+		"import": "./dist/index.mjs",
+		"require": "./dist/index.js"
+	},
+	"sideEffects": false,
+	"scripts": {
+		"build": "tsc -b src",
+		"watch": "tsc -b src --watch",
+		"watch:types": "tsc -b src -w",
+		"lint": "eslint src --ext ts --fix -c ../../package.json",
+		"prepublishOnly": "yarn build"
+	},
+	"dependencies": {
+		"@discordjs/collection": "^0.5.0",
+		"@sapphire/utilities": "3.6.2",
+		"@skyra/http-framework": "workspace:^",
+		"@skyra/i18next-backend": "workspace:^",
```

Comment:
```suggestion
		"@skyra/http-framework": "*",
		"@skyra/i18next-backend": "*",
```

Reply:
Why? This works flawless with Turbo.

Reply:
Because it won't work flawlessly when published to npm.....

Reply:
Then how does Discord.js do it? They use `workspace:^` and the published versions have the correct versions to the other packages.

Reply:
I have no idea but `workspace:^` is gone in the published bundle: https://unpkg.com/browse/discord.js@14.0.0-dev.1649505803-3c0bbac/package.json

Reply:
Exactly, and the actual code uses it: https://github.com/discordjs/discord.js/blob/main/packages/discord.js/package.json#L50-L52